resources/guest_configs/microvm-kernel-ci-x86_64-5.10-no-acpi.config

```diff
@@ -466,48 +475,7 @@ CONFIG_PM_CLK=y
 # CONFIG_WQ_POWER_EFFICIENT_DEFAULT is not set
 # CONFIG_ENERGY_MODEL is not set
 CONFIG_ARCH_SUPPORTS_ACPI=y
-CONFIG_ACPI=y
```

Comment:
That seems wrong. This (and all the rest of ACPI configs) should not be enabled in the previous config for this kernel.

Reply:
I can fold this one back into the previous commit

Reply:
done